src/main/scala/fi/oph/koski/ytr/YtrKoesuoritusServlet.scala

```diff
@@ -16,7 +16,7 @@ class YtrKoesuoritusServlet(implicit val application: KoskiApplication) extends
     val examPaper = getStringParam("copyOfExamPaper")
     val hasAccess = hasAccessTo(examPaper)
     if (koesuoritukset.koesuoritusExists(examPaper) && hasAccess) {
-      contentType = "application/pdf"
+      contentType = if (examPaper.contains("pdf")) "application/pdf" else "text/html"
```

Comment:
En oo ihan varma miten noi tiedostot nimetään, mutta

```suggestion
      contentType = if (examPaper.endsWith(".pdf")) "application/pdf" else "text/html"
```

olisi ehkä vähän vielä parempi tapa hoitaa homma.

Reply:
Testattu toimivaksi tuotannossa. Nimeäminen on muotoa YYYY[K|S]_ID.[pdf|html]